src/System.Net.Http.WinHttpHandler/src/System/Net/Http/WinHttpHandler.cs

```diff
@@ -1632,67 +1108,36 @@ private void SetRequestHandleTlsOptions(SafeWinHttpHandle requestHandle)
 
         private void SetRequestHandleClientCertificateOptions(SafeWinHttpHandle requestHandle, Uri requestUri)
         {
-            // Must be HTTPS scheme to use client certificates.
-            if (requestUri.Scheme != UriSchemeHttps)
+            if (requestUri.Scheme != WinHttpRequestState.UriSchemeHttps)
```

Comment:
Same as above: please use existing code in Common defining the Https scheme string.